### PR TITLE
added question and rules page

### PR DESCRIPTION
**Pull request by pankaj-prog**
Quiz app - questions

Quiz app - rules

### CODE REVIEW

File: src/pages/rules/rules.css [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+  flex-direction: column;
+  align-items: center;
+}
+
+.rules-list {
+  display: flex;
+  flex-direction: column;
+  padding: 16px;
+  gap: 16px;
+  font-size: large;
```
Comment (notmafiaa) [inline src/pages/rules/rules.css:14]: suggestion : can use rems

**Review by kushalsourav — APPROVED** [review]
looks good to merge

File: src/pages/questions/questions.html [hunk 2]
```diff
@@ -0,0 +1,194 @@
...
+          />
+          <label class="answer-label" for="q4-option1">Option 1</label><br />
+          <input
+            class="answer-option-input"
+            type="radio"
+            id="q4-option2"
+            name="ans2"
+            value="option2"
+          />
+          <label class="answer-label" for="q4-option2">Option 2</label><br />
```
Comment (notmafiaa) [inline src/pages/questions/questions.html:143]: can use buttons also

File: src/pages/questions/questions.html [hunk 3]
```diff
@@ -0,0 +1,194 @@
...
+      </h1>
+
+      <div class="question-wrapper">
+        <p class="question">
+          Q.1 Lorem ipsum dolor sit amet consectetur adipisicing elit. Modi,
+          veritatis doloremque! Perspiciatis eaque veritatis odit.
+        </p>
+        <form>
+          <input
+            class="answer-option-input"
```
Comment (notmafiaa) [inline src/pages/questions/questions.html:30]: looks great , naming conventions is superb!!

**Review by notmafiaa — APPROVED** [review]

**Review by Debarshi95 — APPROVED** [review]
Looks good. Can merge.

File: src/pages/questions/question.css [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+  justify-content: space-between;
+  width: 100%;
+  margin-bottom: 1rem;
+}
+.message,
+.timer {
+  border-bottom: 2px solid var(--BLACK);
+}
+
+/* question wrapper */
```
Comment (gauravsinhaweb) [inline src/pages/questions/question.css:26]: remove commented text.

File: src/pages/questions/questions.html [hunk 5]
```diff
@@ -0,0 +1,194 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="/src/pages/questions/question.css" />
+    <title>Document</title>
```
Comment (gauravsinhaweb) [inline src/pages/questions/questions.html]: update the title.

**Review by gauravsinhaweb — APPROVED** [review]
Good to merge, UI looks clean and good.

File: src/pages/rules/rules.css [hunk 6]
```diff
@@ -0,0 +1,15 @@
...
+.rules-wrapper {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.rules-list {
+  display: flex;
+  flex-direction: column;
+  padding: 16px;
```
Comment (ShrutiLakhani) [inline src/pages/rules/rules.css:12]: You may want to change the px to rem here

**Review by ShrutiLakhani — APPROVED** [review]
Clean code and naming conventions are up to the mark. Great Job. Just made a small suggestion. Please check

File: src/pages/questions/questions.html (hunk 5 above)
Comment (pankaj-prog) [reply src/pages/questions/questions.html]: yeah i will

**Pull request merged**